services/blockchain-indexer/methods/dataService/blockchainApps.js

```diff
@@ -29,9 +29,10 @@ module.exports = [
 		name: 'blockchain.apps',
 		controller: getBlockchainApps,
 		params: {
-			chainID: { optional: true, type: 'string' },
+			chainID: { optional: true, type: 'number' },
```

Comment:
Let us keep it as a `string` and add support for a range query in the dataService layer.
```suggestion
			chainID: { optional: true, type: 'string' },
```